branding_company/model/branding_company.py

```diff
@@ -30,6 +29,29 @@ class BrandingCompany(models.Model):
         supporting models and reports.
         """
 
+    def get_partner_branding_company(self, partner_id):
+        """Returns recordset with branding company for partner or False."""
+        if not partner_id:
+            return False
+        partner_model = self.env['res.partner']
+        partner_obj = partner_model.browse(partner_id)
+        return partner_obj and partner_obj.branding_company_id or False
```

Comment:
@hbrunn I think you are mistaken. I always return a RecordSet with either zero or one branding company. It would not be logical to return a filled branding company recordset, or an empty partner recordset.